WooCommerce/Classes/POS/Models/PointOfSaleSettingsService.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+import struct Yosemite.SiteSetting
+import enum Yosemite.SettingAction
+import enum Yosemite.Plugin
+import class Yosemite.PluginsService
+import Observation
+
+@Observable final class PointOfSaleSettingsService {
+    private(set) var receiptStoreName: String?
+    private(set) var receiptStoreAddress: String?
+    private(set) var receiptStorePhone: String?
+    private(set) var receiptStoreEmail: String?
+    private(set) var receiptRefundReturnsPolicy: String?
+    private(set) var isLoading: Bool = false
+    private(set) var shouldShowReceiptInformation: Bool = false
+
+    private var siteID: Int64 {
+        ServiceLocator.stores.sessionManager.defaultSite?.siteID ?? 0
+    }
```

Comment:
I think it's cleaner and safer DI'ing the site ID. When entering POS, the site ID is known and I don't think we want it to change throughout the POS session.